packages/core/conductor/src/workflow/loader.ts

```diff
@@ -134,7 +135,7 @@ export class WorkflowLoader {
     const output = node.outputSchema ? toEffectSchema(node.outputSchema) : AnyOutput;
     const result: Executable = {
       meta: { input: node.inputSchema ? toEffectSchema(node.inputSchema) : AnyInput, output },
-      exec: synchronizedComputeFunction((input) => executeFunction(path, input, output)),
+      exec: synchronizedComputeFunction((input) => executeFunction(path, input[DEFAULT_INPUT], output)),
```

Comment:
doesn't seem right -- DEFAULT_INPUT is for nodes that have just one connection, but the function node has a plug for each function parameter.